src/components/modals/ActionScorer.tsx

```diff
@@ -40,6 +40,38 @@ interface IRenderableColumn extends OF.IColumn {
     render: (actionForRender: ActionForRender, component: ActionScorer, index: number) => React.ReactNode
 }
 
+const findNumberFromMemory = (memory: CLM.Memory, isMultivalue: boolean): number | undefined => {
+    if (isMultivalue) {
```

Comment:
Is multivalue supported yet or prep for when it is?

Reply:
I didn't know what you meant by "supported"?

Single Value number entities use the "number from label"
Multi Value entities use the "number of labels"

Reply:
Asking if work to make multi-value conditions work was also done (looks like it was?)

Reply:
This function could use a comment explaining the behaviour for various entity value types.

Reply:
To answer both:
1. yes I think multi value work was done (depends the cardinality / number of memory values)
2. will add comment